### DISCUSSION TITLE
Moving Next.js app to Blitz.js

### DISCUSSION THREAD

**Discussion by taylorpreston** [blitz-js/blitz#2627] [Q&A]
Hey everyone!

I'm working on a next.js app with prisma, tailwind, and next-auth and it seems like it would be pretty easy to port over to Bliz.js

I'm really looking for any gotcha's or advice in what that process might look like / if it is a bad idea.

Thanks for looking!

**Comment by MrLeebo** [comment]
In theory, I believe you can port a nextjs app to blitz just by changing the next.config.js to blitz.config.js and updating the scripts in package.json, but I'd be curious to know if it turns out that way in practice. Some of the aspects of the "blitz experience" are baked into the new app template, such as the prisma db and the auth module, so if you're hoping to take advantage of that, it may be better to just create a new blitz app and then copy+paste your pages into it.

**Comment by beerose** [comment]
Hey! It's been a while since this discussion started, but we recently added docs on migrating to Blitz: https://blitzjs.com/docs/migrating-to-blitz and maybe it will still be useful to you :)
